code/config.py

```diff
@@ -321,8 +323,15 @@ def renewable_potential_input_paths(paths, param):
 
     paths["TS_ren"] = {}
     pathtemp = paths["region"] + "Renewable energy" + fs + "Regional analysis" + fs + subregions + fs + "Regression outputs" + fs
-    for tech in param["technology"]:
-        quantiles = "_".join(sorted(map(str, param["ren_potential"][tech][0])))
+
```

Comment:
Still not how I imagine it.
I would have written:
paths["TS_ren"] = {
        "WindOn": pathtemp + "long_file_name.csv",
        "WindOff": pathtemp + "other_long_file_name.csv",
 ....
}